src/main/java/io/quarkus/bot/RemoveLabelFromClosedIssue.java

```diff
@@ -0,0 +1,43 @@
+package io.quarkus.bot;
+
+import io.quarkiverse.githubapp.ConfigFile;
+import io.quarkiverse.githubapp.event.Issue;
+import io.quarkus.bot.config.QuarkusBotConfig;
+import io.quarkus.bot.config.QuarkusBotConfigFile;
+import io.quarkus.bot.util.Labels;
+import org.jboss.logging.Logger;
+import org.kohsuke.github.GHEventPayload;
+import org.kohsuke.github.GHIssue;
+import org.kohsuke.github.GHLabel;
+
+import javax.inject.Inject;
+import java.io.IOException;
+
+public class RemoveLabelFromClosedIssue {
+
+   private static final Logger LOG = Logger.getLogger(RemoveLabelFromClosedIssue.class);
+
+   @Inject
+   QuarkusBotConfig quarkusBotConfig;
```

Comment:
You can remove the config file here (and all the code related to it), we don't need it.